src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -195,6 +195,47 @@ private static void VerifyMaxThreads(int expectedMaxw, int expectedMaxc)
         }
 
         [Fact]
+        private static void SetMinThreadsTo0Test()
+        {
+            int minw, minc, maxw, maxc;
+            ThreadPool.GetMinThreads(out minw, out minc);
+            ThreadPool.GetMaxThreads(out maxw, out maxc);
+            void ResetThreadCounts()
+            {
+                Assert.True(ThreadPool.SetMaxThreads(maxw, maxc));
+                Assert.True(ThreadPool.SetMinThreads(minw, minc));
+            }
+
+            try
+            {
+                Assert.True(ThreadPool.SetMinThreads(0, minc));
+                Assert.True(ThreadPool.SetMaxThreads(1, maxc));
+
+                int count = 0;
+                var done = new ManualResetEvent(false);
+                WaitCallback callback = null;
+                callback = state =>
+                {
+                    ++count;
+                    if (count > 100)
+                    {
+                        done.Set();
+                    }
+                    else
+                    {
+                        ThreadPool.QueueUserWorkItem(callback);
+                    }
+                };
+                ThreadPool.QueueUserWorkItem(callback);
+                done.WaitOne(ThreadTestHelpers.UnexpectedTimeoutMilliseconds);
```

Comment:
What are you trying to test here?

Reply:
I'm testing that it makes progress on work items even though the min thread count is set to 0 since that's an allowed value. There was an assertion failure in this case that is fixed as well, so a bit of coverage for that as well.

Reply:
I see.  Ok.